config/settings.yml

```diff
@@ -386,7 +386,7 @@ search:
   url: https://search.usa.gov/api/v2
 
 bing:
-  base_api_url: 'https://dev.virtualearth.net/REST/v1/Routes'
+  base_api_url: 'https://dev.virtualearth.net/REST/v1'
```

Comment:
Is there a corresponding change that needs to be made in production/staging config?

Reply:
The 'dev' is referring to the bing api available to devs, not our dev environment. This link is used in all environments.

Reply:
@edmkitty Ah -- i wasn't clear about what change i meant. 

The `base_api_url` changed here in the settings.yml file, but there are different ways that the `bing.base_api_url` are set in staging and/or production settings files/env that I wanted to make sure got the same change. /cc @whuang85

Reply:
Also @whuang85 dealt with this in the comment below when the `/Routes` got moved